### PR TITLE
Capstone one

### PR DESCRIPTION
**Pull request by billymohajeri**
The capstone project one:
In this milestone, I:

- Added mobile design of the home and about page.
- Added desktop design of the home and about page.
- Added mobile menu for both main and about pages.

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+                >Finnish</a
+              >
+            </ul>
+
+            <ul class="desktop-only white-nav">
+              <li>
+                <a href="./index.html">
+                  <img
+                    src="./assets/logo-black.png"
+                    alt=""
```
Comment (levy002) [inline about.html]: - [ ] Kindly make sure to add alternative text `alt` for all images. :wink:

File: about.html [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+              </li>
+              <li><a href="./index.html#program">Program</a></li>
+              <li><a href="./index.html#see-program">Join</a></li>
+              <li><a href="./index.html#sponsor">Sponsor</a></li>
+              <li><a href="./index.html#news">News</a></li>
+              <li id="bordered-menu-item"><a href="">Buy a ticket</a></li>
+            </ul>
+
+            <section class="mobile-only" id="ham">
+              <img src="./assets/ham.png" alt="" onclick="showMobileMenu()" />
```
Comment (levy002) [inline about.html]: - [ ] Kindly make sure to add alternative text `alt` for all images. :wink:

File: about.html [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+        <h3 class="main-program-title">See the film weeks in Finland</h3>
+        <hr class="logo-part-line1" />
+        <h2 class="about-3-title">
+          Take a look at the last two film weeks<br />which took place in Oulu
+          and Tampere
+        </h2>
+
+        <section class="about-images">
+          <img src="./assets/11.jpg" alt="" class="red-image" />
+          <img src="./assets/22.jpg" alt="" class="red-image" />
```
Comment (levy002) [inline about.html]: - [ ] Kindly make sure to add alternative text `alt` for all images. :wink:

File: about.html [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+      </section>
+
+      <section class="partners-container mobile-only" id="sponsor">
+        <h3 class="main-program-title">Partner</h3>
+        <hr class="main-program-line" />
+        <section class="partners">
+          <img src="./assets/cann.png" alt="" />
+          <img src="./assets/imdb.png" alt="" />
+          <img src="./assets/bbc.png" alt="" />
+          <img src="./assets/oscars.png" alt="" />
```
Comment (levy002) [inline about.html]: - [ ] Kindly make sure to add alternative text `alt` for all images. :wink:

File: about.html [hunk 5]
```diff
@@ -0,0 +1,164 @@
...
+          src="./assets/logo-white.png"
+          alt=""
+          class="footer-about-logo desktop-only"
+        />
+
+        <img
+          src="./assets/logo-black.png"
+          alt=""
+          class="footer-about-logo mobile-only"
+        />
```
Comment (levy002) [inline about.html]: - [ ] Kindly make sure to add alternative text `alt` for all images. :wink:

File: about.html [hunk 6]
```diff
@@ -0,0 +1,164 @@
...
+        <h2 class="footer-about-text">
+          2023 film week in Finland<br />Some rights reserved.
+        </h2>
+      </section>
```
Comment (levy002) [inline about.html]: - [ ] Kindl make sure to wrap your footer in the `footer` tag always in order to structure your HTML semantically correctly. :wink:

File: about.html [hunk 7]
```diff
@@ -0,0 +1,164 @@
...
+        <li class="mobile-menu-items" onclick="hideMobileMenu()">
+          <a href="#sponsor">Sponsor</a>
+        </li>
+        <li><hr /></li>
+        <li class="mobile-menu-items" onclick="hideMobileMenu()">
+          <a href="./index.html#news">News</a>
+        </li>
+        <li><hr /></li>
+      </ul>
+    </section>
```
Comment (levy002) [inline about.html]: - [ ] Kindly make sure to wrap your navigation bar into the `nav` tag in order to correctly structure your HTML semantically. :wink:

File: index.html [hunk 8]
```diff
@@ -4,8 +4,195 @@
...
+        <li class="mobile-menu-items" onclick="hideMobileMenu()">
+          <a href="./about.html#sponsor">Sponsor</a>
+        </li>
+        <li><hr /></li>
+        <li class="mobile-menu-items" onclick="hideMobileMenu()">
+          <a href="#news">News</a>
+        </li>
+        <li><hr /></li>
+      </ul>
+    </section>
```
Comment (levy002) [inline index.html]: - [ ] Kindly make sure to wrap your navigation bar into the `nav` tag in order to correctly structure your HTML semantically. :wink:

File: index.html [hunk 9]
```diff
@@ -4,8 +4,195 @@
...
+      </section>
+      <section class="desktop-only">
+        <section class="partners-container" id="sponsor">
+          <h3 class="main-program-title">Partner</h3>
+          <hr class="speakers-line" />
+          <section class="partners">
+            <img src="./assets/imdb.png" alt="" />
+            <img src="./assets/oscars.png" alt="" />
+            <img src="./assets/cann.png" alt="" />
+            <img src="./assets/bbc.png" alt="" />
```
Comment (levy002) [inline index.html]: - [ ] Kindly make sure to add alternative text `alt` for all images. :wink:

File: index.html [hunk 10]
```diff
@@ -4,8 +4,195 @@
...
+          <img
+            src="./assets/logo-black.png"
+            alt=""
+            class="footer-home-logo"
+          />
+          <h2 class="footer-home-text">
+            Lorem ipsum dolor sit amet consectetur<br />Lorem ipsum dolor sit
+          </h2>
+        </section>
+      </section>
```
Comment (levy002) [inline index.html]: - [ ] Kindl make sure to wrap your footer in the `footer` tag always in order to structure your HTML semantically correctly. :wink:

File: index.html [hunk 11]
```diff
@@ -4,8 +4,195 @@
...
+        </section>
+
+        <section class="footer-home">
+          <img
+            src="./assets/logo-black.png"
+            alt=""
+            class="footer-home-logo"
+          />
+          <h2 class="footer-home-text">
+            Lorem ipsum dolor sit amet consectetur<br />Lorem ipsum dolor sit
```
Comment (levy002) [inline index.html]: - [ ] Kindly make to have the same content in the footer as the one you have in the `about` page footer in order for your website to look professional. :wink:

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hello @billymohajeri   , :wave:
### I'm Levy, your PR reviewer

Good job so far :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._

- [ ] KIndly remember to set up the `es-lint` linter which will check your javascript code on this PR as required. Kindly use [this setup.](https://github.com/microverseinc/linters-config/tree/master/html-css-js) :wink: 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by billymohajeri** [comment]
Hi @levy002,
Thanks for your feedback.
I already have installed es-lint. Please let me know if something is wrong with it.

image

**Comment by levy002** [comment]
Hello @billymohajeri. We only check what the checks (linters must be there as well) run on the PR and there I can not see the `es-lint`. Please double-check again your `linters.yml` file to be sure the `es-lint` is presented there.

image

**Review by nedjwab — APPROVED** [review]
# 🟢 Project Approved! 🏆

## Hi @billymohajeri   👋

Your project is complete! ✔️ 🏁

Your code was very professional 💯

There is nothing else to say other than... _it's time to merge it_ :shipit:
Congratulations! 🎉

 
## Highlights 🌟

- ✔️ Descriptive commmit messsages
- ✔️ Descriptive PR description
- ✔️ No linter Errors
- The code is top-notch! 💯 and have a huge potential.

Cheers and Happy coding!👏👏👏

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.


_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by billymohajeri** [comment]
Dear @nedjwab 
Thanks for your nice feedback 💐👍

**Pull request merged**